lms/static/scripts/frontend_apps/utils/api.ts

```diff
@@ -217,7 +217,7 @@ export function useAPIFetch<T = unknown>(
 
 export type PolledAPIFetchOptions<T> = {
   /** Path for API call */
-  path: string;
+  path: string | null;
```

Comment:
The API has changed here but the implementation hasn't. Did it already handle null paths correctly in all cases?

Reply:
> Did it already handle null paths correctly in all cases?

Yes, as `usePolledAPIFetch` basically forwards the path to `useAPIFetch`, which already expects `string | null`.

I originally forgot to define it nullable here, and didn't realize because when used in `SyncGradesButton`, the hook didn't need to be invoked "conditionally", while now it does, which made me realize about the overlook.